src/agent0/ethpy/base/transactions_test.py

```diff
@@ -0,0 +1,132 @@
+"""Test transaction behavior."""
+
+import logging
+
+import pytest
+from fixedpointmath import FixedPoint
+from hexbytes import HexBytes
+
+from agent0.core.hyperdrive.interactive import ILocalChain, ILocalHyperdrive
+from agent0.ethpy.base.transactions import build_transaction
+
+# we're testing here
+# pylint: disable=too-many-locals
+
+
+@pytest.mark.anvil
+def test_gas_price_base_multiple_explicit(chain: ILocalChain):
+    """Set the gas price base multiple explicitly."""
+    # set up config, hyperdrive, interface, web3, and agent
+    base_fee_multiple = 100
+    regular_config = ILocalHyperdrive.Config()
```

Comment:
why `regular_`? What are you differentiating from?

Reply:
fixed in 680589b9819148a22dbd8efd774899623da1dbe6